### PR TITLE
Decode Morse Code with Ruby

### PR DESCRIPTION
**Pull request by sumairq**
## In this pull request we implemented the following: 

- A method to decode a Morse code character, takes a string parameter and returns the corresponding character in uppercase (e.g. decode_char(".-") returns "A").
- A method to decode an entire word in Morse code, takes a string parameter and returns the string representation. Every character in a word is separated by a single space (e.g. decode_word("-- -.--") returns "MY").
- A method to decode the entire message in Morse code, takes a string parameter, and returns the string representation. Every word is separated by 3 spaces (e.g.

### CODE REVIEW

File: morse.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+end
+
+def decode_sentence(text)
+  new_sentence = ''
+  new_arr = text.split('   ')
+  new_arr.each do |value|
+    new_sentence += "#{decode_word(value)} "
+  end
+  new_sentence
+end
```
Comment (julie-ify) [inline morse.rb]: - [x] There is a trailing space from what this method returns. Kindly add `.strip` to `new_sentence` that is being returned from this method to remove the trailing space. Hint; `new_sentence.strip`

chrome_PJAmHmLZlV

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @sumairq and @denscholar,

Good job so far!

## Highlight! 
- No linter errors ✔️  

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse.rb (hunk 1 above)
Comment (sumairq) [reply morse.rb]: Thank you for the feedback.

**Review by julie-ify — APPROVED** [review]
## ✔️ ✔️ APPROVED

Hi @sumairq and @denscholar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**